### DISCUSSION TITLE
Misleading translation of "XPath expression" for Spanish

### DISCUSSION THREAD

**Discussion by danielaixer** [aclap-dev/video-downloadhelper#115] [Language & Translation]
In the Spanish UI it appears as "File location expression" ("Expresión de ubicación del archivo"), which is horribly misleading!

It wasn't until I pasted one of these _file location expressions_ into ChatGPT that I learned it was an XPath expression.

A more clear an straightforward translation would simply be: "Expresión XPath"

**Comment by jcfrog** [comment]
We take note, thanks for the feedback

**Comment by paulrouget** [comment]
Thank you. This will be fixed in 8.2.1.
